### DISCUSSION TITLE
pango color issue

### DISCUSSION THREAD

**Discussion by mohan43u** [ImageMagick/ImageMagick#4801] [Help] [answered]
Hi, need help to set background color with pango, this is my commandline

<code>
 convert -size 1080x1920 xc:'#00ff00' -gravity center \( -size 480x640 -background '#00ff00' pango:'<span font="Rekha" bgcolor="#00ff00" fgcolor="#ffffff" size="32000" style="italic" weight="bold">'"${text}"'</span>' \) -composite "${workdir}/${counter}.png"
</code>
<br/>
Here, I use '#00ff00' as background, this is suppose to be green, but I get pink color. why this is happening? is this bug?

**Comment by urban-warrior** [comment, marked as answer]
We get a green background with ImageMagick 7.1.0-22 and ImageMagick 6.9.12-32.  What release of ImageMagick are you using?

**Reply by mohan43u** [reply]
I apologize, I have these commandlines

convert -size 1080x1920 xc:green -gravity center \( -size 540x960 -background green pango:'<span bgcolor="green" fgcolor="white" size="40000">வணக்கம்</span>' \) -composite test.png
gst-launch-1.0 multifilesrc location=test.png num-buffers=1 ! pngdec ! videorate ! videoconvert ! videoscale ! video/x-raw,framerate=1/4 ! x264enc ! mp4mux ! filesink location=test.mp4

The first commandline generates proper image with green background and white text, but when I convert that image into a video, I get pink background instead of green, this only happens with green, I get proper ouput for red and blue. So, I was confused that Imagemagick produced back background first, but then realized that it is the gstreamer, which is making issue.

Anyway, thanks for responding to this issue.
